sonarqube/content.md

```diff
@@ -92,6 +92,14 @@ $ docker build --tag=sonarqube-custom .
 $ docker run -ti sonarqube-custom
 ```
 
+### Avoid hard termination of SonarQube
+
+Starting from SonarQube 7.8, SonarQube stops gracefully, waiting for any tasks in progress to finish. Waiting for in-progress tasks to finish can take a large amount of time which the docker does not expect by default when stopping. To avoid having the SonarQube instance killed by the Docker daemon after 10 seconds, it is best to configure a timeout to stop the container with `--stop-timeout`. For example:
+
+```console
+docker run --stop-timeout 3600
```

Comment:
Did you want a `%%IMAGE%%` at the end like line 16? (it'll get autofilled when README.md is generated)

Reply:
Yes. I added it, among other changes